functional/test

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -e
+
+CDIR=$(cd `dirname $0` && pwd)
+USER_ID=${SUDO_UID:-$(id -u)}
+HOME=$(getent passwd "${USER_ID}" | cut -d: -f6)
+
+cd ${CDIR}/../
```

Comment:
Relatives paths will not help when you run scripts from the outside, i.e. `cd ~ ; sudo ./fleet/functional/test`